ukaea/fair-mast-ingestion
# Improve naming consistency across dataset

**Cannot be merged before:** #44

Adds mappings for name consistency across shots and improved naming and structure for most of the datasets we care about for MAST-U and MAST.

To Test:
 - Run some example shots and check for consistency. For example 50900, 48912, and 45066
 - Code review.
